includes/wptt-webfont-loader.php

```diff
@@ -263,15 +263,15 @@ public function get_remote_url_contents() {
 
 			// Early exit if there was an error.
 			if ( is_wp_error( $response ) ) {
-				return;
+				return '';
 			}
 
 			// Get the CSS from our response.
 			$contents = wp_remote_retrieve_body( $response );
 
 			// Early exit if there was an error.
-			if ( is_wp_error( $contents ) ) {
-				return;
+			if ( '' === $contents ) {
+				return '';
 			}
```

Comment:
This should be deleted, `wp_remote_retrieve_body` return the body or an emptry string.